components/introduction.tsx

```
@@ -1,160 +1,114 @@
-                <a
-                  href={footnote.url}
-                  target="_blank"
-                  rel="noopener noreferrer"
-                >
-                  {footnote.text}
+        </div>
+        <div className="space-y-4 h-64 text-left">
+          <p
+            className="text-sm md:text-base leading-relaxed"
+            dangerouslySetInnerHTML={{ __html: content.intro }}
+          />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)